go.mod

```diff
@@ -4,12 +4,12 @@ go 1.15
 
 require (
 	github.com/blang/semver v3.5.0+incompatible
-	github.com/container-storage-interface/spec v1.0.0
+	github.com/container-storage-interface/spec v1.3.0
 	github.com/go-logr/logr v0.2.1 // indirect
 	github.com/golang/protobuf v1.4.3
 	github.com/google/go-cmp v0.5.2
-	github.com/kubernetes-csi/csi-lib-utils v0.6.1
-	github.com/kubernetes-csi/csi-test v1.1.0
+	github.com/kubernetes-csi/csi-lib-utils v0.7.1
+	github.com/kubernetes-csi/csi-test v2.0.0+incompatible
```

Comment:
It looks like theres a v4.0.2 tag and the module named `github.com/kubernetes-csi/csi-test/v4`

`github.com/kubernetes-csi/csi-test/v4 v4.0.1` and referencing the packages as `github.com/kubernetes-csi/csi-test/v4` _may_ make this "compatible" again?

Reply:
_checking_

Reply:
Updated to the latest release of csi-test